docs/source/concepts/architecture/tests/test-sectors.rst

```diff
@@ -0,0 +1,59 @@
+Test Sectors for GeoIPS
+=======================
+
+We use small, pre-determined test sectors for testing. 
+These sectors provide a variety of scene components 
+including coast and land, edge of geostationary disk, day/night/terminator features etc. 
+
+Naming Convention
+-----------------
+
+Test sector plugin names follow this format:
+
+.. code-block::
+
+   test_<satellite>_<proj>_<res>_<edge|nadir>_<day|night|term>_<OPTIONAL:tc|ar|volc>_<YYYYMMDDTHHMMZ>.yaml
```

Comment:
How would you feel about <day|night|terminator>?  It is a few more characters, but it is not obvious what term refers to

Reply:
Works for me!

Reply:
Changed!